index.Rmd

```diff
@@ -1,105 +1,299 @@
+        stretch = "lin", main = "True Color Composite")
+box(col="white")
 ```
+###FALSE COLOR PLOT.The false color composite uses NIR (5) for red, red (4) for green, and green (3) for blue.
 
 
 ```{r}
-data %>% 
-  ggplot(aes(x=x,y=y,col=category))+
-  geom_point()
+#For Earth as example
+par(col.axis="white",col.lab="white",tck=0)
+plotRGB(image1, r = 5, g = 4, b = 3, axes = TRUE, stretch = "lin", main = "False Color Composite")
+box(col="white")
+
+
+#For Mars 
+par(col.axis="white",col.lab="white",tck=0)
+plotRGB(image2, r = 5, g = 4, b = 3, axes = TRUE, stretch = "lin", main = "False Color Composite")
+box(col="white")
+```
+###Indices of WATER
+
+We can use Modified Normalized Difference Water Index [MNDWI=(Green-MIR)/ (Green+MIR)] to separate water from land. This idea is from Reference=https://www.researchgate.net/post/What_is_the_best_band_combination_for_highlighting_the_water_and_soil_together_in_LANDSAT_image2
+The Author of the reference is Saygin Abdikan and is currently working at the Department of Geomatics Engineering, Hacettepe University. The Author researchGate is https://www.researchgate.net/profile/Saygin_Abdikan
+
+The formula is as below:
+[MNDWI=(Green-MIR)/ (Green+MIR)]
+MNDWI <- (image[[2]] - image[[3]])/(image[[2]] + image[[3]]).
+
+To plot the results with ggplot, we convert the raster into a data frame and use geom_tile.
+The plot generated by the codes below show where are located water/ water ice/ volatiles materials the most and we have then know where to land the rovers and start the mining process.
+
+Band4, Band5, and Band6 are use for Earth where Band4 represent the Green band in our case and Band6 represent the MIR band.
+
+For Mars,the formula change little bit because all bands are collected and stored in the same file in Mars Odysee website. False color or black and white could reveal the distribution of water. However, we use the formula:
+
+MNDWI <- (band4)/(1/band9).
+This formula is derived after multiple observations
+
+```{r}
+#For Earth data as example
+MNDWI <- (Band4 - Band6)/(Band4 + Band6)
+
+
+as(MNDWI, "SpatialPixelsDataFrame") %>% 
+  as.data.frame() %>%
+  ggplot(data = .) +
+  geom_tile(aes(x = x, y = y, fill = layer)) +
+  theme(axis.text = element_blank(),
+        axis.ticks = element_blank(),
+        panel.background = element_blank(),
+        panel.grid.minor = element_blank()) +
+  labs(title = "Water or Water ice or Volatiles materials", 
+       x = " ", 
+       y = " ") +
+  scale_fill_gradient(high = "#CEE50E", 
+                      low = "#087F28",
+                      name = "Water or Water ice or Volatiles materials")
+
+#For Mars. 
+#We derived This formula is derived after multiple observations and applied to different data before using it.
+MNDWI <- (band4)/(1/band9)
+#MNDWI <- (band4-band9)/(band4+band9)
+
+
+as(MNDWI, "SpatialPixelsDataFrame") %>% 
+  as.data.frame() %>%
+  ggplot(data = .) +
+  geom_tile(aes(x = x, y = y, fill = layer)) +
```

Comment:
the aspect ratio of this image is stretched.  I suggest using `gplot()` as I mentioned above.  Alternatively, fix the aspect ratio to be 1:1

Reply:
also, if you use `gplot()` you won't need to convert to spatialpixels.